### DISCUSSION TITLE
Which continent has the longest mountain range in the southern hemisphere?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#139] [Galaxy Trivia] [answered]
**Hint:** 🌎

**Comment by Tynab** [comment, marked as answer]
The continent that has the longest mountain range in the southern hemisphere is South America, with the Andes mountain range extending over 7,000 kilometers (4,300 miles) along the western edge of the continent. The Andes run through seven South American countries, including Venezuela, Colombia, Ecuador, Peru, Bolivia, Chile, and Argentina. The highest peak in the Andes is Aconcagua, located in Argentina, which stands at 6,962 meters (22,841 feet) tall.
